protocol/protocol.tex

```diff
@@ -13920,6 +13993,13 @@
         \vspace{1ex}
   \item $\totalDeferredOutput(\BlockHeight) := \ssum{\fs\, \in\, \FundingStreams \;:\; \fsRecipient(\BlockHeight) \;=\; \DeferredPool}{} \fsValue(\BlockHeight)$
 } %nusix
+\nusixone{
+        \vspace{1ex}
+  \item $\totalDeferredInput(\BlockHeight) := \begin{cases}
+           \LockboxDisbursementAmount,&\caseif \BlockHeight = \ZIPTwoSevenOneActivationHeight \\
```

Comment:
I imagine that `\LockboxDisbursementAmount` will get confusing once the ongoing disbursement mechanism is defined. Maybe rename or clarify that this is the NU 6.1 disbursement amount?